Description: Use system Enet instead of bundled version
 This change is explicitly disliked by upstream, however it is preferred by
 Debian policy.
Origin: http://gitorious.org/chakra-packages/games/commit/1aae10c597cf09a281d47570e43c8892531cd9bf
Author: Adrián Chaves Fernández (Gallaecio) <adriyetichaves@gmail.com>

--- a/src/Makefile
+++ b/src/Makefile
@@ -9,7 +9,7 @@
 PLATFORM= $(shell uname -s)
 PLATFORM_SUFFIX=_native
 
-INCLUDES= -Ishared -Iengine -Igame -Ienet/include
+INCLUDES= -Ishared -Iengine -Igame -I/usr/share/enet/include
 
 STRIP=
 ifeq (,$(findstring -g,$(CXXFLAGS)))
@@ -33,7 +33,7 @@
 CLIENT_LIBS= -mwindows -static-libgcc -static-libstdc++ -L$(WINBIN) -L$(WINLIB) -lSDL -lSDL_image -lSDL_mixer -lzlib1 -lopengl32 -lenet -lws2_32 -lwinmm
 else
 CLIENT_INCLUDES= -DINTERFACE -DIRC $(INCLUDES) -I/usr/X11R6/include `sdl-config --cflags`
-CLIENT_LIBS= -Lenet/.libs -lenet -L/usr/X11R6/lib -lX11 `sdl-config --libs` -lSDL_image -lSDL_mixer -lz -lGL
+CLIENT_LIBS= -lenet -L/usr/X11R6/lib -lX11 `sdl-config --libs` -lSDL_image -lSDL_mixer -lz -lGL
 endif
 ifeq ($(PLATFORM),Linux)
 CLIENT_LIBS+= -lrt
@@ -104,7 +104,7 @@
 SERVER_LIBS= -mwindows -static-libgcc -static-libstdc++ -L$(WINBIN) -L$(WINLIB) -lzlib1 -lenet -lws2_32 -lwinmm
 else
 SERVER_INCLUDES= -DSTANDALONE -DMASTERSERVER -DIRC $(INCLUDES)
-SERVER_LIBS= -Lenet/.libs -lenet -lz
+SERVER_LIBS= -lenet -lz
 endif
 SERVER_OBJS= \
 	shared/crypto-standalone.o \
@@ -169,13 +169,13 @@
 	$(STRIP) ../bin/$(APPSERVER).exe
 endif
 else
-client: libenet $(CLIENT_OBJS)
+client: $(CLIENT_OBJS)
 	$(CXX) $(CXXFLAGS) -o $(APPCLIENT) $(CLIENT_OBJS) $(CLIENT_LIBS)
 ifneq (,$(STRIP))
 	$(STRIP) $(APPCLIENT)
 endif
 
-server: libenet $(SERVER_OBJS)
+server: $(SERVER_OBJS)
 	$(CXX) $(CXXFLAGS) -o $(APPSERVER) $(SERVER_OBJS) $(SERVER_LIBS)
 ifneq (,$(STRIP))
 	$(STRIP) $(APPSERVER)
